webui/react/src/components/JupyterLabModal.tsx

```diff
@@ -442,9 +457,9 @@ const JupyterLabForm: React.FC<{
       <Form.Item initialValue={defaults?.name} label="Name" name="name">
         <Input placeholder="Name (optional)" />
       </Form.Item>
-      <Form.Item initialValue={defaults?.pool} label="Resource Pool" name="pool">
-        <Select allowClear placeholder="Pick the best option">
-          {resourcePools.map((pool) => (
+      <Form.Item label="Resource Pool" name="pool">
+        <Select allowClear disabled={!selectedWorkspace} placeholder="Pick the best option">
```

Comment:
nit: i dont think workspace id would take `0` in the future, but as a convention, `selectedWorkspaceId === undefined` can avoid a potential bug.